deegree-ogcapi-features/src/main/resources/META-INF/schemas/ogcapi/features/3.4.0/features.xsd

```diff
@@ -25,6 +25,22 @@
         </element>
         <element name="HtmlViewId" type="string" minOccurs="0"/>
         <element name="Metadata" type="oaf:MetadataType" minOccurs="0"/>
+        <element name="Collection" minOccurs="0" >
+        	<complexType>
+        		<sequence>
+        			<element name="Id" minOccurs="1" type="string"/>
```

Comment:
Done, however, making this `id` instead of `Id` stands this element apart from the convention used in this workspace. All the elements start with a capital letter. Should I still use `id`?

Reply:
You are right, but what I meant was using an XML attribute instead of an XML element.